### DISCUSSION TITLE
Collapse not working on Accunt Pages

### DISCUSSION THREAD

**Discussion by NWierling** [microsoft/fluentui-blazor#1002] [Q&A] [answered]
Hello everyone,
I have created a new project using the new FluentUI template and .NET8. I have set the following settings in the template. 

![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/8de19192-3d2f-4d93-855c-dd76a80cf1d8)

If I now click on the Login NavBar Entry, I cannot collapse the NavBar. I think it has to do with the RenderMode for everything in the Account section. 

![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/f36250cd-487c-4af0-9836-9f496b7b3775)

The ExpandedChanged event is then not fired. Can someone help me with a solution how to toggle it from the Account Pages (Static Mode)?

**Comment by vnbaaij** [comment]
Hi,

Collapsible on the NavMenu is dependant on Blazor running, so it will not work with SSR mode which the Account pages are using because of the code block you posted. 
Is there a specific reason you want to run those pages in SSR mode? If not, I suggest to remove that `RenderModeForPage` method (as you are running on rendermode Server globally anyway)

**Comment by NWierling** [comment]
Hi and thanks for the response :)

![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/0233ced5-ff1f-413a-ab55-bd8cb12c7323)

The code for RenderModeForPage is generated by the template. It is not written by myself. If I remove the code its still not working.

![image](https://github.com/microsoft/fluentui-blazor/assets/136691532/f62d72fe-74ff-40f4-95e3-21f6f0aeeae8)

If I set the render mode for the account pages on Interactive Server, the components reload permanently in a loop

**Comment by vnbaaij** [comment, marked as answer]
Would leaving the acoount pages in SSR mode an setting a `@rendermode=InteractiveServer` attribute on the call to NavMenu in MainLayout work?

**Reply by NWierling** [reply]
That works :) thanks for your help!
